test_scripts/Polices/Validation_of_PolicyTables/Check_count_of_removals_for_bad_behavior_too_many_pending_requests.lua

```diff
@@ -129,7 +130,7 @@ end
 function Test:Check_TOO_MANY_REQUESTS_in_DB()
   local db_path = config.pathToSDL.."storage/policy.sqlite"
   local sql_query = "SELECT count_of_removals_for_bad_behavior FROM app_level WHERE application_id = '" .. config.application1.registerAppInterfaceParams.appID .. "'"
-  local exp_result = 2
+  local exp_result = 1
```

Comment:
@istoimenova, Since is_db_contains() function as _exp_result_ argument requires table _exp_result_ has to be provided as a table